src/service/viewer-impl.js

```diff
@@ -802,6 +802,25 @@ export class Viewer {
   }
 
   /**
+   * Get the fragment from the url or the viewer.
+   * Strip leading '#' in the fragment
+   * @return {!Promise<string|undefined>}
```

Comment:
I saw [this](https://github.com/ampproject/amphtml/blob/master/src/service/viewer-impl.js#L793) so I thought it's fine.